.circleci/config.yml

```diff
@@ -6,8 +6,11 @@ jobs:
 
     steps:
     - checkout
-    - run: sudo pip install codespell
-    - run: codespell --skip=".git,./assets"
+    - run:
```

Comment:
Please do not include this CI change in the `Fixed typos` commit. Can you please move it into the first commit?

Reply:
@axic --- i updated as your request :)